### PR TITLE
Produce relations with hazardsets in decision tree (sources). Based on #44

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Needs https://github.com/GFDRR/thinkhazard_common/pull/16

### CODE REVIEW

File: thinkhazard_processing/decisiontree.py [hunk 1]
```diff
@@ -27,21 +27,23 @@ def apply_decision_tree(dry_run=False):
 
 def clearall_query():
     return '''
+DELETE FROM datamart.rel_hazardcategory_administrativedivision_hazardset;
 DELETE FROM datamart.rel_hazardcategory_administrativedivision;
```
Comment (fvanderbiest) [inline thinkhazard_processing/decisiontree.py:31]: I'm wondering if we could make this automatic, through the use of cascade drop.

**Comment by fvanderbiest** [comment]
I'm not able to review the SQL-related changes but this looks good.

**Comment by pgiraud** [comment]
It looks good to me too.
It deserves tests though.
OK to merge and add tests later.

File: thinkhazard_processing/decisiontree.py (hunk 1 above)
Comment (arnaud-morvan) [inline thinkhazard_processing/decisiontree.py:31]: yes it seems possible with TRUNCATE TABLE

**Comment by pgiraud** [comment]
Merging now.

**Pull request merged**